prboom2/INSTALL

```diff
@@ -41,17 +17,18 @@ Basic Installation
        apt-get install libsdl2-dev libsdl2-net-dev libsdl2-image-dev \
          libpcre3-dev libsdl2-mixer-dev libfluidsynth-dev \
          libportmidi-dev libmad0-dev libdumb1-dev libvorbis-dev
-
+         
+    You will also need `cmake` and `make` if you haven't already got them.
+         
   2. `cd' to the directory containing the PrBoom distribution (the directory
-     this file is in).
+     this file is in, prboom2).
 
-     Type `./configure' to configure PrBoom for your system.  If you're
-     using `csh' on an old version of System V, you might need to type
-     `sh ./configure' instead to prevent `csh' from trying to execute
-     `configure' itself.
-
-     Running `configure' takes awhile.  While running, it prints some
-     messages telling which features it is checking for.
+     Run `cmake . -DCMAKE_BUILD_TYPE=Release` from this folder
+     (You should remove the -DCMAKE... if you wish to get a debug build
```

Comment:
More exactly, you should provide `-DCMAKE_BUILD_TYPE=Debug` for a debug build. I would never recommend anyone to invoke CMake without any `CMAKE_BUILD_TYPE` parameter at all.